### PR TITLE
feat: add occupancy pictogram in stop_date_time sections

### PR DESCRIPTION
**Pull request by woshilapin**
Depends on a few other PR:
- add `departure_occupancy` in https://github.com/hove-io/navitia-proto/pull/190
- add new criteria `occupancy` in https://github.com/hove-io/navitia-proto/pull/190
- support new criteria in `jormungandr` in https://github.com/hove-io/navitia/pull/3861
- forward new field `departure_occupancy` in https://github.com/hove-io/navitia/pull/3862
- support new criteria `occupancy` in https://github.com/hove-io/loki/pull/352
- fill up the `departure_occupancy` in https://github.com/hove-io/loki/pull/357

image

### CODE REVIEW

**Review by pbench — APPROVED** [review]
I don't really know this code so :see_no_evil:

**Review by xlqian — APPROVED** [review]

File: js/pictos.js [hunk 1]
```diff
@@ -131,3 +131,32 @@ pictos.makeEquipmentPicto = function(equipment) {
     }
     return pictos.makeImg(img, equipment);
 };
+
+pictos.makeOccupancyPicto = function(occupancy) {
+    var img = 'OccupancyMedium';
+    var message = 'Unknown';
+    if (occupancy === 'empty') {
```
Comment (xlqian) [inline js/pictos.js:138]: you can use a map for that

Comment (woshilapin) [reply js/pictos.js]: Done for the entire file in [572922b](https://github.com/hove-io/navitia-playground/pull/383/commits/572922bead87ca0353b2310b99fd45a2bcedaa2a).

**Review by xlqian — APPROVED** [review]
:heart:

**Pull request merged**